### PR TITLE
1049 high level users should be able to access photobooth

### PR DESCRIPTION
**Pull request by dudethatbe**
it will render the camera option in the emoji selector if you're logged in and at least level 4.
haven't really tested posting an image yet 😅 it should be posting the base64 image data, but I'll check

### CODE REVIEW

File: app/components/emoji-selector/camera.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import Component from "@ember/component";
```
Comment (mcfiredrill) [inline app/components/emoji-selector/camera.ts:1]:
```suggestion
import Component from "@glimmer/component";
```

File: app/components/emoji-selector/camera.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import { action } from "@ember/object";
+
+export default class CameraMenu extends Component {
+  args: any;
+
+  @action
+  willRender() {
```
Comment (mcfiredrill) [inline app/components/emoji-selector/camera.ts:8]:
```suggestion
  didInsert() {
```

and then add {{did-insert this.didInsert}} to the template

File: app/components/emoji-selector/camera.hbs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<div class="border-solid border-4 border-df-green">
```
Comment (mcfiredrill) [inline app/components/emoji-selector/camera.hbs:1]:
```suggestion
<div {{did-insert this.didInsert}} class="border-solid border-4 border-df-green">
```

**Comment by dudethatbe** [comment]
Ok thanks for the suggestions, now there's nothing but green checkmarks.
I just tried posting a photo and it didn't send a pic so I'll fix that next 😵🥴

**Review by mcfiredrill — COMMENTED** [review]
good

**Pull request merged**